resource-timing/initiator-type/misc-script.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Resource Timing initiator type: extra script types</title>
+<link rel="help" href="https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-initiatortype"/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resource-timing/resources/observe-entry.js"></script>
+<script src="resources/initiator-type-test.js"></script>
+
+</head>
+
+<script>
+  const moduleWorkerURL = 'resources/empty.js?id=moduleWorker';
+  const modulePreloadURL = 'resources/empty.js?modulePreload';
+  const workerURL = 'resources/empty.js?worker';
+  const link = document.createElement('link');
+  link.rel = 'modulepreload';
+  link.href = modulePreloadURL;
+  document.head.appendChild(link);
```

Comment:
Could we move this test case over to `resource-timing/intitiator-type/link.html`? We hope to keep `initiator-type` tests organized by their element.